src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs

```diff
@@ -56,6 +56,13 @@ public interface IInteractsWithApps : IExecuteMethod
         /// <param name="appPath">a string containing the id of the app.</param>
         void RemoveApp(string appId);
 
+        /// <summary>
+        /// Terminates an App.
+        /// </summary>
+        /// <param name="appId">a string containing the id of the app.</param>
+        /// <param name="timeout">an integer number containing the retry-timeout</param>
```

Comment:
this timeout means how long the application should take to be fully terminated. If it needs more time then the function will return false (500ms by default)

Reply:
https://github.com/appium/java-client/blob/6d408e8011e3e81f6d23015876d81760b496f60e/src/main/java/io/appium/java_client/android/appmanagement/AndroidTerminateApplicationOptions.java